app/controllers/submissions_controller.rb

```diff
@@ -8,15 +8,15 @@ def create
   end
 
   def show
-    @submission = Submission.find(params[:id]).includes(:answers)
+    @submission = Submission.find(params[:id])
   end
 
-  def delete
-    @submissions = Form.find(params[:form_id]).submissions
-    if @submissions.nil?
-      render json: {message: "Cannot find submissions"}, status: :not_found
+  def destroy
```

Comment:
Do you now have both `submissions` and `answers`? If so, when you delete a `submission, you also need to delete all corresponding `answers`